app/workers/emails_users_export_worker.rb

```diff
@@ -0,0 +1,40 @@
+require 'csv'
+
+class EmailsUsersExportWorker
+  include Sidekiq::Worker
+  include DumpCommons
+  include DumpEmails
+
+  sidekiq_options queue: :data_low
+
+  attr_reader :export_type
+
+  def perform(export_type=:global)
+    @export_type = export_type
+    begin
+      emails_to_csv_file
```

Comment:
Why is this using inheritance instead of e.g. a generic object that takes an ActiveRecord query scope as input, and dumps it?